templates/python/const.go

```diff
@@ -0,0 +1,8 @@
+package python
+
+const constTpl = `{{ $r := .Rules }}
+{{ if $r.Const }}
+	if {{ accessor . }} != {{ lit $r.GetConst }}:
+		{{ err .}}
```

Comment:
Can you include an error message here, something like [this](https://github.com/envoyproxy/protoc-gen-validate/blob/1c0e3fb8004bf1906f8c398428329541de3aae2f/templates/goshared/const.go#L6)?